src/writeData/components/PluginAddToExistingConfiguration/Customize.tsx

```diff
@@ -0,0 +1,167 @@
+// Libraries
+import React, {ChangeEvent, FC, useEffect} from 'react'
+import {connect, ConnectedProps} from 'react-redux'
+
+// Components
+import {
+  ComponentSize,
+  Form,
+  Grid,
+  Input,
+  InputType,
+} from '@influxdata/clockface'
+import TelegrafConfig from 'src/telegrafs/components/TelegrafConfig'
+
+// Actions
+import {
+  addTelegrafPlugins,
+  setTelegrafConfigName,
+  setTelegrafConfigDescription,
+} from 'src/dataLoaders/actions/dataLoaders'
+
+// Types
+import {AppState, ConfigurationState, ResourceType, Telegraf} from 'src/types'
+import {PluginConfigurationStepProps} from 'src/writeData/components/PluginAddToExistingConfiguration/Wizard'
+
+// Selectors
+import {getAll} from 'src/resources/selectors'
+import {getDataLoaders} from 'src/dataLoaders/selectors'
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = PluginConfigurationStepProps & ReduxProps
+
+const CustomizeComponent: FC<Props> = props => {
+  const {
+    onAddTelegrafPlugins,
+    onSetTelegrafConfigDescription,
+    onSetTelegrafConfigName,
+    pluginConfig,
+    pluginConfigName,
+    setIsValidConfiguration,
+    setPluginConfig,
+    telegrafConfig,
+    telegrafConfigDescription,
+    telegrafConfigName,
+  } = props
+
+  const handleError = error => {
+    setIsValidConfiguration(false)
+    setPluginConfig(`${error}`)
+    console.error(error)
```

Comment:
I'm trying to get us away from `console.error` in favor of a notification to the user that's actionable. Is it reasonable to have a notification here?

Reply:
No reason. It gets caught. But I thought it would be easier to spot an error if someone happens to be debugging. I can remove it.